src/Microsoft.Azure.SignalR.AspNet/ServerConnections/ServiceConnectionManager.cs

```diff
@@ -117,6 +117,20 @@ public virtual Task<bool> WriteAckableMessageAsync(ServiceMessage serviceMessage
             return _appConnection.WriteAckableMessageAsync(serviceMessage, cancellationToken);
         }
 
+        public Task StartGetServersPingAsync()

```

Comment:
is an `Endpoint` level `Start` enough that there is no need to expose this method to the manager?

Reply:
For a normal endpoint start, there's no need to get servers ping to sync status, do you suggest to make it common for all?